rosdep/python.yaml

```diff
@@ -8765,6 +8769,7 @@ python3-scipy:
 python3-scp:
   debian: [python3-scp]
   fedora: [python3-scp]
+  nixos: [python39Packages.scp]
```

Comment:
Should this be ``python3Packages.scp`` like the others added here, instead of ``python39`` specifically ? Ref : https://search.nixos.org/packages?channel=22.05&from=0&size=50&sort=relevance&type=packages&query=Rtree

Reply:
I'm not familiar NixOS packaging system, but since I can't find `python3Packages.*` entries anywhere, I'll go ahead and assume the `python3Packages.` prefix/namespace/group can be used when it is non-ambiguous ie. when there's only one package released. Is that true @kjeremy? Wouldn't that expose us to breakage?

Reply:
Good catch! Fixed. Yes `python3Packages` is an alias.

Reply:
@hidmic looks like it is available : https://search.nixos.org/packages?channel=22.05&show=python310Packages.scp&from=0&size=50&sort=relevance&type=packages&query=scp